### PR TITLE
Add Sumologic to Terraform

### PR DESCRIPTION
**Pull request by chefsalim**
Update Terraform to add the Sumologic agent to nodes (only builder-workers for this change, more will be added later).

Signed-off-by: Salim Alam <salam@chef.io>

tenor-86562023

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: terraform/instances.tf [hunk 1]
```diff
@@ -254,6 +254,11 @@ resource "null_resource" "datastore_provision" {
   }
 
   provisioner "file" {
+    content     = "${data.template_file.sumo_sources_worker.rendered}"
+    destination = "/home/ubuntu/sumo_sources_worker.json"
```
Comment (reset) [inline terraform/instances.tf]: I think you want to write this to `/hab/svc/sumologic/files/builder-worker.json`. We hard configure [syncSource](https://github.com/habitat-sh/core-plans/blob/master/sumologic/config/user.properties#L4) to be the files directory of the sumologic service

Comment (chefsalim) [reply terraform/instances.tf]: Yeah I thought about doing that write directly.  The current design is that we'll use a 'hab file upload' to populate it into that directory, so if we make any changes to the config, we can push up that change like any other config.

Comment (elliott-davis) [reply terraform/instances.tf]: oh nice! Carry on!

**Pull request merged**